test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -206,6 +207,7 @@ public void before() throws Exception {
         
         participant = new StudyParticipant.Builder()
                 .withRoles(CALLER_ROLES)
+                .withSubstudyIds(ImmutableSet.of("substudyA"))
```

Comment:
Minor: This test inconsistently uses teh CALLER_SUBS constant or creates a new ImmutableSet on the fly.

Reply:
Cleaned this up